src/ui/app_view_model.py

```
@@ -31,24 +34,24 @@ def __init__(self):
         self.enrollment_session_queue = []
         self.current_enrollment_student = None
         self.capture_step = 0
+        self.capture_step_image_count = 0  # New: count images per step
+        self.CAPTURE_IMAGES_PER_STEP = 5   # New: configurable images per step
         # --- NEW, MORE DETAILED PROMPTS ---
         self.capture_prompts = [
             "CAPTURE COMPLETE", # Index 0
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)